src/safe_apps/serializers.py

```diff
@@ -35,11 +35,18 @@ def to_representation(self, instance: Tag) -> str:
         return instance.name
 
 
+class FeatureSerializer(serializers.Serializer[Feature]):
+    @staticmethod
+    def to_representation(instance: Feature) -> str:
+        return instance.key
+
+
 class SafeAppsResponseSerializer(serializers.ModelSerializer[SafeApp]):
     id = serializers.IntegerField(source="app_id")
     provider = ProviderSerializer()
     access_control = serializers.SerializerMethodField()
     tags = serializers.SerializerMethodField()
+    features = serializers.SerializerMethodField()
```

Comment:
This will expose `features` once this hits staging and then production. If there are no concerns in its release from your side then I think it's good to go. If you consider this as something that is work in progress we can hide this behaviour with a feature flag (so that you can test on staging freely without affecting the API exposed by production).

Reply:
the data format has been agreed and we do not expect any changes in the near future, so i think it's good to go without hiding